src/App.tsx

```diff
@@ -20,17 +17,114 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    itemWidth: 130,
+    frameSize: 3,
+    step: 3,
+    animationDuration: 1000,
+    infinite: false,
   };
 
   render() {
     const { images } = this.state;
+    const { itemWidth } = this.state;
+    const { frameSize } = this.state;
+    const { step } = this.state;
+    const { animationDuration } = this.state;
+    const { infinite } = this.state;
+
+    const changeStep = (e: ChangeEvent<HTMLInputElement>) =>
+      this.setState({ step: +e.target.value });
```

Comment:
```suggestion
    const handleChange = (e: ChangeEvent<HTMLInputElement>) =>
      this.setState({ [e.target.name]: +e.target.value });
```
Let's make 1 shared `handleChange` function (at least for number inputs, but you can also handle boolean too - check input type or name). Above is an example
You can add `name` attribute to `input` element and use it here. Note that some additional checks might be necessary to make TypeScript happy :)